src/test/scala/se/nimsa/sbx/dicom/streams/DicomStreamOpsTest.scala

```diff
@@ -83,29 +89,29 @@ class DicomStreamOpsTest extends TestKit(ActorSystem("DicomStreamOpsSpec")) with
     val bytes = preamble ++ fmiGroupLength(unsupportedMediaStorageSOPClassUID ++ tsuidExplicitLE) ++ unsupportedMediaStorageSOPClassUID ++ tsuidExplicitLE ++ patientNameJohnDoe
     val source = StreamSource.single(bytes)
     recoverToSucceededIf[DicomStreamException] {
-      source.runWith(dicomStreamOpsImpl.dicomDataSink(storage.fileSink("name"), storage.parseFlow(None), (_, _) => Future.successful(Seq.empty), Contexts.imageDataContexts, reverseAnonymization = true))
+      source.runWith(dicomStreamOpsImpl.dicomDataSink(storage.fileSink("name"), storage.parseFlow(None), (_, _) => Future.successful(Seq.empty), Contexts.imageDataContexts, reverseAnonymization = true, tagFilterFlow = dicomStreamOpsImpl.NO_ACTION_FILTER))
```

Comment:
With default arguments as discussed above the addition  `tagFilterFlow = dicomStreamOpsImpl.NO_ACTION_FILTER)` can be left out.